docs/monitors/collectd-systemd.md

```diff
@@ -0,0 +1,107 @@
+<!--- GENERATED BY gomplate from scripts/docs/monitor-page.md.tmpl --->
+
+# collectd/systemd
+
+This [SignalFx Smart Agent](https://github.com/signalfx/signalfx-agent) monitor collects metrics about the state of
+configured systemd services using the [collectd-systemd](https://github.com/signalfx/integrations/tree/master/collectd-systemd)
+plugin. A service is in the state that a metric represents if the metric value is 1 and not in the state if the
+metric value is 0. The name of the service is assigned to dimension `plugin_instance` of the reported metric.
+
+Below is a sample SignalFx Smart Agent YAML configuration for monitoring the state of `docker` and `ubuntu-fan`
+services. See [here](https://github.com/signalfx/signalfx-agent#configuration) for SignalFx Smart Agent
+configuration details.
+```yaml
+monitors:
+- type: collectd/systemd
+  intervalSeconds: 10
+  services:
+    - docker
+    - ubuntu-fan
+```
+Only metric `gauge.substate.running` which indicates whether a service is running or not gets reported by default.
+The other metrics must be configure explicitly (see below). Also, it is recommended that you include the extra
+dimension `metric_source` with a meaningful value in order to facilitate filtering in the SignalFx app.
+```
+monitors:
+- type: collectd/systemd
+  intervalSeconds: 10
+  services:
+    - docker
+    - ubuntu-fan
+  extraDimensions:
+    metric_source: collectd/systemd
+```
+
+
+Monitor Type: `collectd/systemd`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/collectd/systemd)
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `services` | **yes** | `list of strings` | Systemd services to report on |
+
+
+
+
+## Metrics
+
+The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
+
+| Name | Type | Included | Description |
+| ---  | ---  | ---    | ---         |
+| `gauge.active_state.active` | gauge |  | Indicates that the systemd unit/service is active |
```

Comment:
Where do this other non-included metrics come from? I don't see them emitted by the plugin.

Reply:
If you add them to the plugin, make them enabled/disabled by config flags only.  Make the default configuration of the plugin only emit the single metric you have marked as "included".